lib/project_types/script/layers/domain/script_json.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Script
+  module Layers
+    module Domain
+      class ScriptJson
+        attr_reader :filename, :content, :version, :title, :description, :configuration_ui, :configuration
+
+        def initialize(filename:, content:)
+          @filename = filename
+          @content = content
```

Comment:
Mostly convenience. Theres a few places (like [here](https://github.com/Shopify/shopify-app-cli/pull/1276/files#diff-b1b4783accd40805bb28b1e948f47e6db4881cedea88ecb59b389450defac702R152)) where we access `content` instead of manually grabbing every key-value pair or reading the file again.